app/controllers/webauthn_verifications_controller.rb

```diff
@@ -35,6 +42,17 @@ def authenticate
     session.delete(:webauthn_authentication)
   end
 
+  def status
```

Comment:
Can be done in a follow up: I really like the idea of redirecting to another page for the result! Something that irks me a bit is that anyone could just go to this page without even going through the verification process and they are always accessible.

Is there a way so the page is only shown when appropriate?

Reply:
Hmm yeah we could address this in a follow up if we'd like. My thought process here was, if a user navigates to this page independently, it doesn't give them anything, they're not actually logged in. Additionally, the page isn't linked anywhere so a user is unlikely to stumble across it. So to me it didn't seem worth it to do anything more complex like saving the state of the last response to the CLI to protect the route.

Reply:
> Additionally, the page isn't linked anywhere so a user is unlikely to stumble across it.

That's a good point! Hmm this makes me think that we need to make sure that search engines don't put this in their index so it won't be surfaced.

> if a user navigates to this page independently, it doesn't give them anything, they're not actually logged in.

I agree it doesn't provide value and doesn't pose a security risk, but it's surfacing false information which imo should be addressed at some point of time.

Reply:
it sounds like storing a flag in the session is what we'd be looking for here?

Reply:
To store it in the rails session, we'd have to set it in the controller, before the javascript callback lets us know if it was actually a success or failure. So to Jenny's point about "surfacing false information," we still wouldn't know if it was an actual success or failure.

We could alternatively set this in a cookie. The downside of that is that cookie's are easily changeable by the user, but since it's not a security concern that's probably fine. However, I'm running into trouble trying to delete the cookie after it's used once (it deletes and then magically reanimates on page refresh), so as initially discussed I'm going to save this for a follow up PR to unblock this one.